src/config/config.ts

```
@@ -69,7 +69,8 @@ export function basePickerConfig(): PickerConfigInternal {
     skinTonesDisabled: false,
     suggestedEmojisMode: SuggestionMode.FREQUENT,
     theme: Theme.LIGHT,
-    width: 350
+    width: 350,
+    unicodeToHide: [],
```

This is where we'll add the default emojis to hide.